manifests/base/code-server-notebook-imagestream.yaml

```diff
@@ -6,8 +6,8 @@ metadata:
     opendatahub.io/notebook-image: "true"
   annotations:
     opendatahub.io/notebook-image-url: "https://github.com/red-hat-data-services/notebooks/tree/main/codeserver"
-    opendatahub.io/notebook-image-name: "Code server"
-    opendatahub.io/notebook-image-desc: "Code server workbench allows users to code, build, and collaborate on projects directly from web."
+    opendatahub.io/notebook-image-name: "code-server"
+    opendatahub.io/notebook-image-desc: "code-server workbench allows users to code, build, and collaborate on projects directly from web."
```

Comment:
@jstourac collaborate? Do you know what feature this refers to specifically? Can more than one person access the code-server at the same time and use the web ui?

Reply:
all the workbenches can be accessed at the same time if the owner of the workbench provides access to the user.
example:
if multiple people have access to a DS project.
they all can use all the workbenches spined up in the DS project.